src/main/java/com/softserve/edu/sporthubujp/service/impl/ArticleServiceImpl.java

```diff
@@ -37,4 +36,24 @@ public void deleteArticleById(String id)
         }
         articleRepository.deleteById(id);
     }
+
+    public Article updateArticle(Article newArticle, String id)
+    {
+        return articleRepository.findById(id)
+            .map(article -> {
```

Comment:
you can use MapStruct like:

`void update(@MappingTarger Article articleFromDb, Article newArticle);`